common/src/Model/ChangeBrushFaceAttributesRequest.h

```diff
@@ -134,14 +134,14 @@ namespace TrenchBroom {
             void setSurfaceFlags(int surfaceFlags);
             void unsetSurfaceFlags(int surfaceFlags);
             void replaceSurfaceFlags(int surfaceFlags);
-            void setSurfaceFlag(size_t surfaceFlag);
-            void unsetSurfaceFlag(size_t surfaceFlag);
+            void setSurfaceFlag(int surfaceFlag);
```

Comment:
Hmm, on second thought I feel really uneasy about this. What does a negative index mean? This should remain size_t.

Reply:
Yeah, bitfields are the one thing that should really be unsigned. Sorry, I know tweaking this stuff is annoying